### ISSUE TITLE
VE Scenario with modified categories

### ISSUE THREAD

**Issue by dsrosenthal5** [VisionEval/VisionEval-Dev#53]
Can VEScenario be run with modified categories and/or scenario inputs that do not match those in the VERSPM default files? I am able to run the multi-runs and the html window for the Viewer opens but no charts appear, only the headers.
![image](https://user-images.githubusercontent.com/48492597/60120558-ec001500-9735-11e9-99b6-9ee86bbfb8e0.png)

**Comment by dflynn-volpe** [comment]
I ran into this myself when testing out the visualizer, when using a simple set of scenarios where not all the  
categories had levels. I have some notes on [this Google Sheet ](https://docs.google.com/document/d/1BlBCVAW48sOlAtdeFFmcQPYQBRa1B0gsQ6K92avkMj0/edit#) from my experiment. 

The Scenario Viewer is going to read from five JSON files, in `VEScenarioViewer/data/VERSPM`. Your categories need to be consistent across them. Can you share these five files (`category-cfg.js`, `output-cfg.js`, `scenario-cfg.js`, `verspm.js`, and `verspm-ref.js`)? 

I suspect you have some inconsistency between them, and also that because you have only two categories of scenarios (Diversion and Roadways), your outputs don't match up with what `verspm.html` is expecting. Namely, `verspm.html` has hardcoded category abbreviations, which is fine if you are using the default categories (B for Bicycles, for example), but not if you have your own categories.

Note that you can also get the outputs directly from `verspm.js`.

**Comment by dsrosenthal5** [comment]
@dflynn-volpe I took a quick look but notice anything in the files - I'll read through that document and see if can figure it out. In the meantime, here are the output files:

[VERSPM.zip](https://github.com/VisionEval/VisionEval-Dev/files/3331016/VERSPM.zip)

**Comment by dflynn-volpe** [comment]
I made a manual adjustment to `verspm.html` to deal with just two categories -- attached is the first attempt. Basically, I've commented out anything that referred to more than two scenario categories (search for `/*` to find the comments) and manually changed the names of the categories to return to be 'B' and 'R', to match your categories.

We should be able to make the .html file more flexible to account for a number of scenario categories different from 6, and to read from the category-cfg.js to set up what the category names should be. This should work in the meantime. Let me know if this works for you, and we can work on making the scenario viewer more flexible.

[verspm.zip](https://github.com/VisionEval/VisionEval-Dev/files/3331089/verspm.zip)
